common/grouping/grouping-en.html

```diff
@@ -0,0 +1,243 @@
+<dl>
+	<div>
+		<dt> Last modified time </dt>
+		<dd> 2004-12-23T23:33Z </dd>
+	</div>
+	<div>
+		<dt> Recommended update interval </dt>
+		<dd> 60s </dd>
+	</div>
+	<div>
+		<dt> Authors </dt>
+		<dt> Editors </dt>
+		<dd> Robert Rothman </dd>
+		<dd> Daniel Jackson </dd>
+	</div>
+</dl>
+
+<p> The following example shows the dl element used to give a set of instructions. The order of the instructions here is important (in the other examples, the order of the blocks was not important). </p>
+<p>Determine the victory points as follows (use the
+first matching case):</p>
+<dl>
+	<dt> If you have exactly five gold coins </dt>
+	<dd> You get five victory points </dd>
+	<dt> If you have one or more gold coins, and you have one or more silver coins </dt>
+	<dd> You get two victory points </dd>
+	<dt> If you have one or more silver coins </dt>
+	<dd> You get one victory point </dd>
+	<dt> Otherwise </dt>
+	<dd> You get no victory points </dd>
+</dl>
+
+<p> The following snippet shows a dl element being used as a glossary. Note the use of dfn to indicate the word being defined. </p>
+<dl>
+	<dt><dfn>Apartment</dfn>, n.</dt>
+	<dd>An execution context grouping one or more threads with one or
+	more COM objects.</dd>
+	<dt><dfn>Flat</dfn>, n.</dt>
+	<dd>A deflated tire.</dd>
+	<dt><dfn>Home</dfn>, n.</dt>
+	<dd>The user's login directory.</dd>
+</dl>
+
+
+<pre><code>&lt;p&gt; Example 1 &lt;/p&gt;
+&lt;dl&gt;
+	&lt;dt&gt; .. &lt;/dt&gt;
+	&lt;dd&gt; .. &lt;/dd&gt;
+	&lt;dd&gt; .. &lt;/dd&gt;
+	...
+&lt;/dl&gt;
+
+
+&lt;p&gt; Example 2 &lt;/p&gt;
+&lt;dl&gt;
+	&lt;dt lang="en-US"&gt; &lt;dfn&gt;color&lt;/dfn&gt; &lt;/dt&gt;
+	&lt;dt lang="en-US"&gt; &lt;dfn&gt;color&lt;/dfn&gt; &lt;/dt&gt;
+	&lt;dd&gt; ... &lt;/dd&gt;
+&lt;/dl&gt;
+
+&lt;p&gt; Example 3 &lt;/p&gt;
+&lt;dl&gt;
+	&lt;div&gt;
+		&lt;dt&gt; .. &lt;/dt&gt;
+		&lt;dd&gt; .. &lt;/dd&gt;
+	&lt;/div&gt;
+	&lt;div&gt;
+		&lt;dt&gt; .. &lt;/dt&gt;
+		&lt;dd&gt; .. &lt;/dd&gt;
+	&lt;/div&gt;
+&lt;/dl&gt;
+
+&lt;p&gt; Example 4 &lt;/p&gt;
+&lt;dl&gt;
+	&lt;dt&gt; .. &lt;/dt&gt;
+	&lt;dd&gt; .. &lt;/dd&gt;
+	&lt;dt&gt; .. &lt;/dt&gt;
+	&lt;dd&gt; .. &lt;/dd&gt;
+	...
+&lt;/dl&gt;
+
+&lt;p&gt; Example 5 &lt;/p&gt;
+&lt;dl&gt;
+	&lt;dt&gt;&lt;dfn&gt; .. &lt;/dfn&gt; .. &lt;/dt&gt;
+	&lt;dd&gt; .. &lt;/dd&gt;
+	&lt;dt&gt;&lt;dfn&gt; .. &lt;/dfn&gt; .. &lt;/dt&gt;
+	&lt;dd&gt; .. &lt;/dd&gt;
+	...
+&lt;/dl&gt;</code></pre>
+
+<h2>The <code>figure</code> element</h2>
+<figure>
+	<img src="pic_trulli.jpg" alt="Trulli" style="width:100%">
+	<figcaption>Fig.1 - Trulli, Puglia, Italy.</figcaption>
+</figure>
+<figure>
+	<p>'Twas brillig, and the slithy toves<br>
+	Did gyre and gimble in the wabe;<br>
+	All mimsy were the borogoves,<br>
+	And the mome raths outgrabe.</p>
+	<figcaption><cite>Jabberwocky</cite> (first verse). Lewis Carroll, 1832-98</figcaption>
+</figure>
```

Comment:
Add an example where the figcaption is the first child element

Like:
```suggestion
<figure>
	<p>'Twas brillig, and the slithy toves<br>
	Did gyre and gimble in the wabe;<br>
	All mimsy were the borogoves,<br>
	And the mome raths outgrabe.</p>
	<figcaption><cite>Jabberwocky</cite> (first verse). Lewis Carroll, 1832-98</figcaption>
</figure>
<figure>
	<figcaption><cite>Jabberwocky</cite> (first verse). Lewis Carroll, 1832-98</figcaption>
	<p>'Twas brillig, and the slithy toves<br>
	Did gyre and gimble in the wabe;<br>
	All mimsy were the borogoves,<br>
	And the mome raths outgrabe.</p>
</figure>

```